versioned/tiered/src/main/java/com/dremio/nessie/tiered/builder/DynamoL1Builder.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.tiered.builder;
+
+import java.util.Map;
+
+public class DynamoL1Builder implements L1Builder<Map<String, AttributeValue>> {
```

Comment:
Nit: the per-store L1 builders should be in the store-implementations and rather not be `public`